src/main/java/de/tum/in/www1/artemis/web/rest/SystemNotificationResource.java

```diff
@@ -134,11 +134,11 @@ public SystemNotificationResource(SystemNotificationRepository systemNotificatio
 
     /**
      * GET /system-notifications/:id : get the "id" system notification.
+     * This route is also accessible for unauthenticated users.
      *
      * @return the ResponseEntity with status 200 (OK) and with body the notification, or with status 404 (Not Found)
      */
     @GetMapping("/system-notifications/active-notification")
-    @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
     public SystemNotification getActiveSystemNotification() {
         log.debug("REST request to get active SystemNotification : {}");
```

Comment:
```suggestion
        log.debug("REST request to get active SystemNotification");
        return systemNotificationService.findActiveSystemNotification();
```
Just since you already modified this class ;)